### DISCUSSION TITLE
New Addons for 1.12.2

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#763] [Compatibility Addon Suggestions]
To make it easier for everyone, I will be compiling all mod compat suggestions for 1.12.2 on this post going forward.
From now on all 1.12.2 mod support suggestion should go as replies to this post.
For 1.16.5 suggestions, please refer to the [1.16.5 suggestion list](https://github.com/supermassimo/DynamicTrees-ExC/issues/59).
Suggestions from the [discord server](https://discord.gg/83qcpkY) will also be added here.

**Please please please include how many trees a mod adds when suggesting**

> DO NOT suggest the following types of mods as they will not be considered:
>  - OTG mods
>  - Mods without a public source (such as MCreator mods)

Those in **bold** were considered and/or are being worked on.
When a mod is finished and published, it is removed from this list.

The stars are a rough estimate of how likely it is for me (subjectively, anyone is free to pick any!) to choose that mod.
It is based on how popular the mod is, how many trees it has, how unique these are and how interesting I find them.
**0 stars just means i do not know enough about the mod, not that i do not like it**.

- ★★★★☆[**AbyssalCraft**](https://www.curseforge.com/minecraft/mc-mods/abyssalcraft) (2 trees)
- ★★☆☆☆[Advent Of Ascension](https://www.curseforge.com/minecraft/mc-mods/advent-of-ascension-nevermine) (20 trees)
- ★★★☆☆[**Aether II**](https://www.curseforge.com/minecraft/mc-mods/the-aether-ii) (11 trees)
- ★☆☆☆☆[Ice and Fire](https://www.curseforge.com/minecraft/mc-mods/ice-and-fire-dragons) (1 log)
- ★★★☆☆[Environs](https://www.curseforge.com/minecraft/mc-mods/environs) (8 trees)
- ★★★★☆[Redwoods](https://www.curseforge.com/minecraft/mc-mods/redwoods) (1 tree)
- ★☆☆☆☆[Wild Nature](https://www.curseforge.com/minecraft/mc-mods/wildnature) (a bjillion trees)
- ★☆☆☆☆[Binnie's Trees](https://www.curseforge.com/minecraft/mc-mods/binnies-mods) (a bjillion trees)
- ★★★☆☆[Better with Addons](https://www.curseforge.com/minecraft/mc-mods/better-with-addons) (3 trees)
- ★★★☆☆[Fossils and Archeology](https://www.curseforge.com/minecraft/mc-mods/fossils) (4 trees)
- ★★☆☆☆[Simply Tea](https://www.curseforge.com/minecraft/mc-mods/simply-tea) (1 tree)
- ★★☆☆☆[Divine RPG](https://www.curseforge.com/minecraft/mc-mods/official-divinerpg) (11 trees)
- ★★☆☆☆[Novam Terram](https://www.curseforge.com/minecraft/mc-mods/novam-terram) (59 trees)
- ☆☆☆☆☆[Auxiliary Biomes](https://www.curseforge.com/minecraft/mc-mods/auxiliary-biomes) (2 biomes)
- ☆☆☆☆☆[Wizardry Mod](https://www.curseforge.com/minecraft/mc-mods/wizardry-mod) (1 tree)
- ☆☆☆☆☆[Gaia Dimension](https://www.curseforge.com/minecraft/mc-mods/gaia-dimension) (9 trees)
- ★★★★☆[Project Vibrant Journeys](https://www.curseforge.com/minecraft/mc-mods/project-vibrant-journeys) (16 trees)
- ★★★☆☆[Spooky Biomes](https://www.curseforge.com/minecraft/mc-mods/spooky-biomes) (4 trees)
- ★★★☆☆[Primal Core](https://www.curseforge.com/minecraft/mc-mods/primalcore) (3 trees)
- ★★★☆☆[**The Erebus**](https://www.curseforge.com/minecraft/mc-mods/primalcore) (8 trees)
- ☆☆☆☆☆[Calculator](https://www.curseforge.com/minecraft/mc-mods/calculator) (4 trees)
- ★★★☆☆[The Aurorian](https://www.curseforge.com/minecraft/mc-mods/the-aurorian) (2 trees)
- ★☆☆☆☆[Stygan End](https://www.curseforge.com/minecraft/mc-mods/stygian-end-biome-expansion) (1 tree)
- ★☆☆☆☆[Rainbow Oak Trees](https://www.curseforge.com/minecraft/mc-mods/rainbow-oak-trees) (1 tree)
- ☆☆☆☆☆[Growthcraft](https://www.curseforge.com/minecraft/mc-mods/growthcraft-community-edition) (3 trees)
- ☆☆☆☆☆[Mo' Creatures](https://www.curseforge.com/minecraft/mc-mods/mo-creatures) (2 trees)
- ☆☆☆☆☆[More Biomes XL](https://www.curseforge.com/minecraft/mc-mods/morebiomesxl) (3 trees)
- ☆☆☆☆☆[The Midnight](https://www.curseforge.com/minecraft/mc-mods/the-midnight) (5 trees)
- ☆☆☆☆☆[Chinjunfu](https://www.curseforge.com/minecraft/mc-mods/chinjufumod) (4 trees)
- ★★★☆☆[Biodiversity Hotspot](https://www.curseforge.com/minecraft/mc-mods/ani-mod-biodiversity-hotspot) (1 tree)
- ★★★★☆[Tropicraft](https://www.curseforge.com/minecraft/mc-mods/tropicraft) (6 trees)
- ☆☆☆☆☆[Glacidus](https://www.curseforge.com/minecraft/mc-mods/glacidus) (2 trees)
- ★★☆☆☆[Blue Skies](https://www.curseforge.com/minecraft/mc-mods/blue-skies) (4 trees)
- ☆☆☆☆☆[Journey into the Light](https://www.curseforge.com/minecraft/mc-mods/journey-into-the-light-mod) (13 trees)
- ★★☆☆☆[Warriors Path](https://www.curseforge.com/minecraft/mc-mods/warriors-path) (2 trees)
- ★★★★★[Misty World](https://www.curseforge.com/minecraft/mc-mods/misty-world) (13 trees)
- ★★★★☆[Beasts](https://www.curseforge.com/minecraft/mc-mods/beasts) (1 tree)
- ★★★☆☆[Tofucraft](https://www.curseforge.com/minecraft/mc-mods/tofucraftreload) (2 trees)
- ☆☆☆☆☆[Prodigy Tech](https://www.curseforge.com/minecraft/mc-mods/prodigy-tech) (1 tree)
- ★★★★☆[Zoesteria](https://www.curseforge.com/minecraft/mc-mods/zoesteria) (8 trees)
- ☆☆☆☆☆[Explorercraft](https://www.curseforge.com/minecraft/mc-mods/explorercraft-worldexpansion) (5 trees)
- ☆☆☆☆☆[Natural Pledge](https://www.curseforge.com/minecraft/mc-mods/natural-pledge) (28 trees)
- ☆☆☆☆☆[The Creeping Nether](https://www.curseforge.com/minecraft/mc-mods/the-creeping-nether) (1 tree)
- ☆☆☆☆☆[Dartcraft](https://www.curseforge.com/minecraft/mc-mods/dartcraft-reloaded) (1 tree)
- ★☆☆☆☆[Oh The Biomes You'll Go](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go)  (45 trees)
- ☆☆☆☆☆[Prehistoric Trees](https://www.curseforge.com/minecraft/mc-mods/prehistoric-trees) (8 trees)
- ☆☆☆☆☆[Greg Tech Community Edition](https://www.curseforge.com/minecraft/mc-mods/gregtechce) (1 tree)
- ★★★☆☆[Nethercraft Classic](https://www.curseforge.com/minecraft/mc-mods/nethercraft-classic) (1 tree)
- ☆☆☆☆☆[Tirphycraft](https://www.curseforge.com/minecraft/mc-mods/tirphycraft) (3 trees)
- ☆☆☆☆☆[Pokecube AIO](https://www.curseforge.com/minecraft/mc-mods/pokecube-aoi) (12 trees)
- ★★★★☆[Terraqueous](https://www.curseforge.com/minecraft/mc-mods/terraqueous) (10 trees)
- ★★★★☆[EvilCraft](https://www.curseforge.com/minecraft/mc-mods/evilcraft) (1 tree)
- ★★★★☆[Douglas Forest](https://www.curseforge.com/minecraft/mc-mods/douglas-forest) (3 trees)
- ★★☆☆☆[Apricon Trees](https://www.curseforge.com/minecraft/mc-mods/apricorn-tree-farm)
- ★★★★☆[Ancient Spellcraft](https://www.curseforge.com/minecraft/mc-mods/ancient-spellcraft) (1 tree)
- ☆☆☆☆☆[Kathairis](https://www.curseforge.com/minecraft/mc-mods/beta-kathairis) (many trees)
- ★★★☆☆[Wasteland Reborn](https://www.curseforge.com/minecraft/mc-mods/wasteland-reborn) (1 tree)
- ☆☆☆☆☆[Jurassic World Reborn](https://www.curseforge.com/minecraft/mc-mods/jurassic-world-reborn-mod) (5 trees)
- ☆☆☆☆☆[Immersive intelligence](https://www.curseforge.com/minecraft/mc-mods/immersive-intelligence) (1 tree)
- ☆☆☆☆☆[Potato Power Craft](https://www.curseforge.com/minecraft/mc-mods/potato-power-craft) (1 tree, 1 dimension)
- ☆☆☆☆☆[Into the Maelstorm](https://www.curseforge.com/minecraft/mc-mods/into-the-maelstrom) (3 trees)
- ☆☆☆☆☆[Scape and Run: Parasites](https://www.curseforge.com/minecraft/mc-mods/scape-and-run-parasites) (1 tree, 3 variants)
- ☆☆☆☆☆[Rusticated Fruit](https://www.curseforge.com/minecraft/mc-mods/rusticated-fruit) (3 trees)
- ★☆☆☆☆[Ultimate Backport Mod](https://www.curseforge.com/minecraft/mc-mods/ultimatebackportmod) (4 trees)

If you continuously suggest mods that are already in the list multiple times, you will be blocked from participating in this discussion.

**Comment by TheConception** [comment]
As a request:

IC2 Exp has a popular fork called IC2 Classic. Current addon for IC2 Exp can't be used with IC2 Classic as it has its own implementation for rubber tree (you also cant load both IC2 Exp and IC2 Classic at the same time). It's literally the same tree with same mechanics but just with different IDs and stuff. It shouldn't be really hard to make a separate version for the IC2 Classic fork.

**Comment by DizzyDJW** [comment]
Rubber trees from TechReborn?
I do believe they are the only trees in that mod

**Comment by supermassimo** [comment]
> Rubber trees from TechReborn?
> I do believe they are the only trees in that mod

I have already done Tech Reborn https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-tech-reborn

**Comment by DizzyDJW** [comment]
> > Rubber trees from TechReborn?
> > I do believe they are the only trees in that mod
> 
> I have already done Tech Reborn https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-tech-reborn

Ah, thank you kindly, it didn't show up in the search under "Dynamic Trees"

**Comment by disastercube** [comment]
Biome Bundle trees. 
The mod ads a lot of different new tree ( some bigger or smaller then normal).
It be nice for it to be compatible with it and with its tree sizes (meaning new tree models).

**Comment by supermassimo** [comment]
> Biome Bundle trees.
> The mod ads a lot of different new tree ( some bigger or smaller then normal).
> It be nice for it to be compatible with it and with its tree sizes (meaning new tree models).

Please read the post before commenting, OTG mods are not supported.

**Comment by ZotyDev** [comment]
GregTech Community Edition.
Only 1 tree, the Rubber Tree, and it has the same structure of small vanilla oak trees.

**Comment by WenXin20** [comment]
[Pokecube AIO](https://www.curseforge.com/minecraft/mc-mods/pokecube-aoi) Support
It adds several trees that produce berries, trees are only grown when the corresponding berries are planted on farmland.

**Comment by Nathan22211** [comment]
is it possible tree tweaker support could be done this mod allows you to make your own trees

**Comment by rubencillo900** [comment]
Terraqueus.
It adds 10 fruit trees 
curseforge page: https://www.curseforge.com/minecraft/mc-mods/terraqueous

There is another mod called Realworld mod which is the succesor of plant mega pack, and adds many trees, but, now is discontinued and i dont know if his creator, 10paktimbits keeps the archives because the website is no longer active.

**Comment by Gonbuto** [comment]
Please support Heat and Climate.
https://www.curseforge.com/minecraft/mc-mods/heat-and-climate
This mod contains date, morus, olive, lemon and walnut trees.
I know Dynamic Trees - Heat and Climate Compat, but this mod hasn't corresponded now  Heat and Climate.

Sorry, I'm not good at English.

**Comment by ghost** [comment]
[Quark](curseforge.com/minecraft/mc-mods/quark)
Please Support it. Blossom Leaves Inside.

**Comment by Harleyoc1** [comment]
@VarielForCode There’s already an [addon for quark](https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-quark).

**Comment by supermassimo** [comment]
> Please make compatibility mods for;
> Stygian End,The Erebus and Defiled Lands and maybe for recurrent complex?

The Erebus and Defiled Lands are already being worked on. Please read the post before suggesting

**Comment by Atopik2** [comment]
Erebus and Stygian End
Tree numbers are already written but stygian end only adds 1 tree please make a mod for that 🙏

**Comment by supermassimo** [comment]
> Erebus and Stygian End
> Tree numbers are already written but stygian end only adds 1 tree please make a mod for that 🙏

Stygian End's tree is too big to be made dynamic. And again, both Stygian End and Erebus is already on the list.

**Comment by Atopik2** [comment]
Can you make stygian trees in ahape of willow and in size of that giant
redwood/speuces that willblook pretty good please.

6 Kas 2020 Cum 01:05 tarihinde supermassimo <notifications@github.com> şunu
yazdı:

**Comment by Harleyoc1** [comment]
> Can you make stygian trees in ahape of willow and in size of that giant redwood/speuces that willblook pretty good please. 6 Kas 2020 Cum 01:05 tarihinde supermassimo <notifications@github.com> şunu yazdı:
> […](#)
> Erebus and Stygian End Tree numbers are already written but stygian end only adds 1 tree please make a mod for that 🙏 Stygian End's tree is too big to be made dynamic. And again, Erebus is already on the list. — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#41 (comment)](https://github.com/supermassimo/DynamicTrees-ExC/issues/41#issuecomment-722672841)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARUSFW5FGCJRSQDDQ33MSALSOMOSFANCNFSM4Q3BCNZA> .

We will not add trees to addons unless we can respect how they are in the original mod, or failing that how they are in reality. The Stygian End trees are too large to be made dynamic, it would ruin the whole look of the biome to have smaller trees.

**Comment by Atopik2** [comment]
Thanks i understand why you cant make stygian end compatible with dynamic
trees;
Can you make compatibility between giant mushrooms from vanilla minecraft
and the ones at the twillight forest and a compatibility for defiled
lands,thanks :)

8 Kas 2020 Paz 14:29 tarihinde Harleyoc1 <notifications@github.com> şunu
yazdı:

**Comment by Harleyoc1** [comment]
> Thanks i understand why you cant make stygian end compatible with dynamic trees; Can you make compatibility between giant mushrooms from vanilla minecraft and the ones at the twillight forest and a compatibility for defiled lands,thanks :) 8 Kas 2020 Paz 14:29 tarihinde Harleyoc1 <notifications@github.com> şunu yazdı:
> […](#)

Defiled lands is already on the list, and dynamic mushrooms has been suggested as an idea in DynamicTreesTeam/DynamicTrees-ExC#28. Please make sure you look at the list before making suggestions (you could use `ctrl/cmd + f` to search for a mod).

**Comment by SlimefrogSocial** [comment]
1 vote more for The Midnight, Stygian (I read the other comments but it is still on the list so I will still vote), & The Creeping Nether

**Comment by supermassimo** [comment]
> 1 vote more for The Midnight, Stygian (I read the other comments but it is still on the list so I will still vote), & The Creeping Nether

All of those are already listed.

**Comment by supermassimo** [comment]
> Hello. Could you add compability to Aether: Lost Content mod https://www.curseforge.com/minecraft/mc-mods/aether-lost-content ? It doesn't add any new trees but it adds a dungeon that has crystal trees outside it and those trees arent dynamic trees even if you have aether addon installed while the gold dungeon has dynamic trees outside it. https://puu.sh/GMFJg/ff37e71922.png

The Aether addon already has a certain compatibility with Lost Content. if its installed, you will be able to craft and plant crystal trees. Unfortunately i was unable to remove the crystal trees from the dungeon, as these are hardcoded into its generation.

**Comment by DarkWolfe5** [comment]
I'm looking for Nature's Aura mod compatibility.for 1.12.2
When replaceVanillaSapling = true the Ritual of the Forest don't want to work.
When I place either an oak acorn or a vanilla sapling, it turns into a DT sapling, which NA doesn't recognize as satisfying the ritual's need for an oak sapling.

I already have Dynamic Trees - Nature's Aura by HarleyOConnor installed, and that doesn't fix anything about the tree rituals.

From my digging in the code of both NA and DT, the NA check is trying to match the center block with instanceof BlockSapling or BlockLog. (Found in /naturesaura/blocks/multi/Multiblocks.java)
I thought changing the extend of BlockDynamicSapling in DT to BlockSapling would work, but that build just crashes the client on loading.
I've tried to add the DT saplings and branches to the ritual check, but that has no effect so far, but I could be doing it wrong. I'm not very savvy in writing Java, let alone MC mods.

I've also filed a report/request (#167) with Nature's Aura. I'll update here if I get a response over there.

Nature's Aura: https://www.curseforge.com/minecraft/mc-mods/natures-aura
DT-Nature's Aura: https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-natures-aura

**Comment by Harleyoc1** [comment]
> I'm looking for Nature's Aura mod compatibility.for 1.12.2
> When replaceVanillaSapling = true the Ritual of the Forest don't want to work.
> When I place either an oak acorn or a vanilla sapling, it turns into a DT sapling, which NA doesn't recognize as satisfying the ritual's need for an oak sapling.
> 
> I already have Dynamic Trees - Nature's Aura by HarleyOConnor installed, and that doesn't fix anything about the tree rituals.
> 
> From my digging in the code of both NA and DT, the NA check is trying to match the center block with instanceof BlockSapling or BlockLog. (Found in /naturesaura/blocks/multi/Multiblocks.java)
> I thought changing the extend of BlockDynamicSapling in DT to BlockSapling would work, but that build just crashes the client on loading.
> I've tried to add the DT saplings and branches to the ritual check, but that has no effect so far, but I could be doing it wrong. I'm not very savvy in writing Java, let alone MC mods.
> 
> I've also filed a report/request (#167) with Nature's Aura. I'll update here if I get a response over there.
> 
> Nature's Aura: https://www.curseforge.com/minecraft/mc-mods/natures-aura
> DT-Nature's Aura: https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-natures-aura

I never got around to implementing the ritual of the forest yet - it states this clearly on the CurseForge page. I never thought about `replaceVanillaSapling` messing that up so that’s an error on my part, is it something you need to work or could you turn it off for the time being? I will release a patch to the mod and remove the sapling replacer for the time being anyway.

**Comment by DarkWolfe5** [comment]
> I never got around to implementing the ritual of the forest yet - it states this clearly on the CurseForge page. I never thought about `replaceVanillaSapling` messing that up so that’s an error on my part, is it something you need to work or could you turn it off for the time being? I will release a patch to the mod and remove the sapling replacer for the time being anyway.

"Need" is always a subjective term. ;-) I can get away without this function, it just won't look as pretty as I intend in the long run. Bonus points if I could assign specific ages of the tree before the ritual kicks off, depending on the recipe (but that wouldn't be your dept. anyways).

I can turn off the replaceVanillaSapling setting for now and wait patiently, and hopefully, for this detail to be added.

Thank you for the quick reply!

P.S. I looked through the curseforge page, and didn't see anything about dynamic trees and anything in NA, other than a link to the DT-NA addon. *shrug* No biggie... I HAVE been going blind lately though... dang aging!! B-)

**Comment by supermassimo** [comment]
> > I never got around to implementing the ritual of the forest yet - it states this clearly on the CurseForge page. I never thought about `replaceVanillaSapling` messing that up so that’s an error on my part, is it something you need to work or could you turn it off for the time being? I will release a patch to the mod and remove the sapling replacer for the time being anyway.
> 
> "Need" is always a subjective term. ;-) I can get away without this function, it just won't look as pretty as I intend in the long run. Bonus points if I could assign specific ages of the tree before the ritual kicks off, depending on the recipe (but that wouldn't be your dept. anyways).
> 
> I can turn off the replaceVanillaSapling setting for now and wait patiently, and hopefully, for this detail to be added.
> 
> Thank you for the quick reply!
> 
> P.S. I looked through the curseforge page, and didn't see anything about dynamic trees and anything in NA, other than a link to the DT-NA addon. _shrug_ No biggie... I HAVE been going blind lately though... dang aging!! B-)

This post is to suggest new mods for compatibility, for any other suggestion or bug report please open a new issue.

**Comment by Btmy520** [comment]
Hello, I want to ask how worldgen is written. I want to generate Pam’s tree. Thank you！

**Comment by LEWeee** [comment]
> ★★★☆☆Sakura (5 trees)

Sakura apparently was supposed to have built in Dynamic Trees support at one point 0999312/Sakura_mod#44.

Response from the main dev:
> 目前放弃对动态树的支持
实在是过于繁琐，无法下笔。

Translated (at the courtesy of Google Translate):
> Support for dynamic trees is currently abandoned
It is too cumbersome to write.

Just thought it was worth mentioning.

Sakura also adds 6 trees, not 5 (Red Maple, Yellow Maple, Orange Maple, Green Maple, Sakura and Plum), 7 if you include Bamboo

**Comment by tyon2006** [comment]
Hey, hoping for support for original Kathairis. Has 4 trees, but I think only 3 would be candidates because of a mushroom decorator. Link for the mod below: 

https://www.curseforge.com/minecraft/mc-mods/beta-kathairis

**Comment by barttraeger** [comment]
Hi there, this one is quite tricky, I believe:
Is it somehow possible to make a compat for the mod Maple Syrup ?
(https://www.curseforge.com/minecraft/mc-mods/maple-syrup)
It's a tree tapping system with custom config options which tree gives which fluid, which is great for modpack developing.
Has this Stardew Valley vibe about it which is really neat.
Problem is the tree tap block from the mod must be placed against a log, which is not possible with dynamic trees.

However, in the Dynamic Trees PHC Addon it's possible to tap maple trees by clicking the trunk with an iron ingot.
Would it be possible to apply this system to configurable trees with configurable fluid output and amount ?
Thanks for any ideas and thank you for all your work!

**Comment by kaguroyume** [comment]
Hey could you add support for Plant Mega Pack? Aside from adding many other kinds of flowers, crops, swamp plants, bushes and shrubs, this mod adds a variety of new trees--52 to be exact. Some big, some small, some short, some tall. Many are fruit trees. It adds coconut and other palm trees. It has to be one of the best tree mods. 
http://www.9minecraft.net/plant-mega-pack-mod/

**Comment by LEWeee** [comment]
> Hey could you add support for Plant Mega Pack? Aside from adding many other kinds of flowers, crops, swamp plants, bushes and shrubs, this mod adds a variety of new trees--52 to be exact. Some big, some small, some short, some tall. Many are fruit trees. It adds coconut and other palm trees. It has to be one of the best tree mods.
> http://www.9minecraft.net/plant-mega-pack-mod/

Plant Mega Pack seems to unfortunately be discontinued as the official website for the mod is offline and the link you provided is from a third party site that often has outdated versions which may or may not even pose a threat to the safety of your computer, so I doubt Plant Mega Pack support will be done any time soon, if a all.

Which is kind of a shame though, I would certainly like to do something similar. It was very useful for decoration as well as breathe some new life into the world without neccessarily adding any new biomes.

**Comment by SlimefrogSocial** [comment]
2 things: 
1 DragonFire mod support would be nice, they only have 2 trees and only 1 has leaves so hopefully it's pretty simple (it's a Patreon mod for now {Beta release} which I have, can't give it out for public for free but for development should be separate from that)
2 I noticed that the projects area hasn't been updated in a while, life just getting busy?

EDIT: Didn't see the rule about public source, I might be able to get it if you want to look into it but I would have to ask for it, it isn't "public" in that sense, sadly... I was really hoping for that mod to have Dynamic Trees as well, but I guess that's just my luck unless it gets chosen and they want to give me the open source version...

**Comment by LEWeee** [comment]
> 1 DragonFire mod support would be nice, they only have 2 trees and only 1 has leaves so hopefully it's pretty simple (it's a Patreon mod for now {Beta release} which I have, can't give it out for public for free but for development should be separate from that)
 
> EDIT: Didn't see the rule about public source, I might be able to get it if you want to look into it but I would have to ask for it, it isn't "public" in that sense, sadly... I was really hoping for that mod to have Dynamic Trees as well, but I guess that's just my luck unless it gets chosen and they want to give me the open source version...

Even if you are able to get the source code, it has to be under such a license that allows for modifying the code as long as certain criteria are met (e.g.: you can't make money out of the creation you use the modified code for, or you can only modify the code if you give credit somewhere to the original creator, etc.), which considering the mod is a Patreon mod the license most likely isn't going to be met (because if it was at that point anyone could hypothetically just take the source code, compile it themselves and give it out for free). I think you would have a much bigger luck asking the original devs for built-in Dynamic Trees support as even if you were able to get the source code the license wouldn't allow us to do anything with it.

**Comment by Harleyoc1** [comment]
> 2 I noticed that the projects area hasn't been updated in a while, life just getting busy?

This is mainly because we are busy working on the 1.16 port, once it’s done add-ons will be the main priority again.

**Comment by Knight9910** [comment]
Not sure where to ask this. Wondering if there's any way to make Dynamic Trees compatible with Mystcraft. Mystcraft doesn't add any new tree types, but it does involve world generation, and apparently uses its own generation system to do it instead of the normal world gen system, which results in Mystcraft dimensions always spawning vanilla trees instead. I was wondering if anyone knew a way to make the two work together.

**Comment by Cr4fTJuNkiE** [comment]
Only 1 Star for Oh The Biomes You Go?! That mod is GREAT! Every bit as good as BOP. Please consider adding more stars! Especially when DT for 1.16.5 is out! Thank you!

**Comment by Harleyoc1** [comment]
> Only 1 Star for Oh The Biomes You Go?! That mod is GREAT! Every bit as good as BOP. Please consider adding more stars! Especially when DT for 1.16.5 is out! Thank you!

@Cr4fTJuNkiE This thread is for **1.12**. We will likely never make an add-on for BYG for 1.12 because it's extremely buggy and I believe unfinished. 

However, for 1.16 it will almost definitely happen. In fact one of the developers of BYG has shown interest in making it himself.

**Comment by Harleyoc1** [comment]
Also, to re-iterate how the stars work: 

`The stars are a rough estimate of how likely it is for me (subjectively, anyone is free to pick any!) to choose that mod.
It is based on how popular the mod is, how many trees it has, how unique these are and how interesting I find them.
0 stars just means i do not know enough about the mod, not that i do not like it.`

**Comment by Cr4fTJuNkiE** [comment]
Aah. OK. Apologies. Didn't see that! Great news!! Thank you.

**Comment by Nathan22211** [comment]
the 1.12 version of BYG, from looking at it with biometweaker is also an MCreator mod

**Comment by Cr4fTJuNkiE** [comment]
Yeah. I was. The dev ditched MCreator and codes with Forge now.

**Comment by Cr4fTJuNkiE** [comment]
*IT

**Comment by kittylegitcheck** [comment]
Biomes o plenty for 1.16.5 pls

**Comment by CrazyDinoLover** [comment]
[Suggestion] Jurassic World Reborn mod (Basicly JurassiCraft but with more trees)

**Comment by yggdrasil75** [comment]
is there possibility of support for future versions/nether backport fungi?

**Comment by DarthCaesium** [comment]
[Advanced Rocketry](https://www.curseforge.com/minecraft/mc-mods/advanced-rocketry) - 3 trees

**Comment by ghost** [comment]
[Suggestion]

Husbandry and Pies! (Forge) for 1.12.2
https://www.curseforge.com/minecraft/mc-mods/hsbdry

It adds a new biome named Mud Field. The mod adds too a new crops, seeds, food and a new entity called mud pig.

**Comment by Harleyoc1** [comment]
> [Suggestion]
> 
> Husbandry and Pies! (Forge) for 1.12.2
> https://www.curseforge.com/minecraft/mc-mods/hsbdry
> 
> It adds a new biome named Mud Field. The mod adds too a new crops, seeds, food and a new entity called mud pig.

How many trees does it add/what is support needed for?

**Comment by ghost** [comment]
> > [Suggestion]
> > Husbandry and Pies! (Forge) for 1.12.2
> > https://www.curseforge.com/minecraft/mc-mods/hsbdry
> > It adds a new biome named Mud Field. The mod adds too a new crops, seeds, food and a new entity called mud pig.
> 
> How many trees does it add/what is support needed for?

It doesn't adds new trees, but on the biome 'mud field' doesn´t generate dynamic trees

**Comment by BIOH4RD** [comment]
When are you gonna update the Thaumcraft compability mod?

**Comment by joe-vettek** [comment]
> 
Most of DT Sakura's work has been completed.

**Comment by joe-vettek** [comment]
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-for-sakura

**Comment by mewj** [comment]
https://www.curseforge.com/minecraft/mc-mods/rusticated-fruit

Planting the tree seeds from this mod grows normal trees instead of dynamic ones. I tried fixing this issue by changing the recipe output using CraftTweaker but it doesn't appear to be compatible, even though the mod it's dependent on, Rustic, is.

**Comment by LUEZIS** [comment]
Mod suggestion: Rusticated Fruit (Rustic addon)
https://www.curseforge.com/minecraft/mc-mods/rusticated-fruit
3 trees included:
- peach tree
![image](https://user-images.githubusercontent.com/40040593/167411060-6479bf30-ffd4-4785-adcd-ccef2591993d.png)
- persimmon tree
![image](https://user-images.githubusercontent.com/40040593/167411090-40b72420-4aa3-494b-b675-0df89061969c.png)
- pear tree
![image](https://user-images.githubusercontent.com/40040593/167411126-55760ac3-cc04-4510-91cc-ffade7eae460.png)
The mod is compatible with BoP fruit types.

**Comment by stealthusk** [comment]
Dynamic Trees for Oh The Biomes You'll Go for 1.12.2?
https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go/files/all?filter-game-version=2020709689%3A6756

Please?

**Comment by stealthusk** [comment]
Dynamic Trees for Recurrent Complex Trees? (I think not possible)
https://www.curseforge.com/minecraft/mc-mods/recurrent-complex

**Comment by Harleyoc1** [comment]
> Dynamic Trees for Oh The Biomes You'll Go for 1.12.2? https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go/files/all?filter-game-version=2020709689%3A6756
> 
> Please?

BYG in 1.12 was never truly finished and known to be very buggy. Aside from that it is too much work when not many people actually play it in that version. As such we have no plans for a back port.

**Comment by Harleyoc1** [comment]
> Dynamic Trees for Recurrent Complex Trees? (I think not possible)
> https://www.curseforge.com/minecraft/mc-mods/recurrent-complex

There isn't really much need for this. The tree structures it adds can be disabled in their config.

**Comment by Joshua444454** [comment]
Add support for Project:Vibrant Journeys

**Comment by Lonelyxiya** [comment]
How do I disable Swamp Oak and Blossoming spawning in the Quark configuration?

**Comment by Prince285** [comment]
Make a DT compat addon for UBM (Ultimate Backport Mod) which adds basically every new feature in 1.13-1.19 to 1.12.2. https://www.curseforge.com/minecraft/mc-mods/ultimatebackportmod

**Comment by Rumcajz23th** [comment]
Hi the Dynamic Trees - Thaumcraft Addons is outdated. The mod Thaumic Bases Unofficial have newer version which is incopatible with this addon. It is possible to make newer version please?

**Reply by supermassimo** [reply]
We have dropped support for our 1.12 addons, but if anyone fixes the issue and makes a PR I would gladly publish an update

**Comment by Kai2315** [comment]
can you update the aether

**Comment by mitchngwen** [comment]
If you can make an addon for [Zoesteria](https://www.curseforge.com/minecraft/mc-mods/zoesteria). I will donate at least $20. I need 1.12.2 in particular. (I do not like the newer versions of minecraft)
